### DISCUSSION TITLE
Ungroup Control Center Taskbar Icons.

### DISCUSSION THREAD

**Discussion by I-Dude** [valinet/ExplorerPatcher#3326] [Ideas]
Hello all.
I use the Windows 10 Taskbar because I dont like that the W11 System functions are grouped up, cant be reordered, and require extra clicks to change settings. Also I use Ear Trumpet for fine tuning of audio devices control so having 2 sound icons is annoying. As is clear in Microsoft tearing out the W10 taskbar code in 24H2, they dont want us to use it....

So I propose folding the existing Control Center into just the Battery Icon and introducing new Taskbar Icons for Network and Sound with custom menus. (all the the menus and icons are already existing, just buried a few clicks deeper.

W10 sound menu
![W10 Taskbar Audio](https://github.com/valinet/ExplorerPatcher/assets/1747168/f8825517-86f5-411a-9ba7-b05dd4fa1971)

EarTrumpet menu
![W10 Taskbar EarTrumpet](https://github.com/valinet/ExplorerPatcher/assets/1747168/3b7449c0-09a2-4eea-ae68-a03f337dde58)

W11 sound menu combined (proposed)
![W11 Taskbar Audio 1](https://github.com/valinet/ExplorerPatcher/assets/1747168/4772e375-097f-4520-bc4a-a6c4b181e6fa)

But that may be too many functions in one menu, so alternatively:
W11 Sound menu (proposed Volume + Mixer) + (proposed Output Selection or use existing Sound Output menu)
![W11 Taskbar Audio 2](https://github.com/valinet/ExplorerPatcher/assets/1747168/4d068e45-efb7-430b-b44b-0d494007ec72)
The system sound icon in the volume mixer kinda sucks/is confusing, maybe use a different icon?

W10 Network menu:
![W10 Taskbar Network](https://github.com/valinet/ExplorerPatcher/assets/1747168/063df7ec-755b-48d4-85d8-1c309f55eeac)

And for the Network menu, just stack the VPN over the WIFI just like in W10
W11 Network menu (proposed):
![W11 Taskbar Network](https://github.com/valinet/ExplorerPatcher/assets/1747168/7bf04866-8c5d-48b6-9a1a-90eb018f6acd)

For completeness, the Battery Icon would just pull up the Control Center
![W11 Taskbar Battery](https://github.com/valinet/ExplorerPatcher/assets/1747168/a3f8e180-ac1c-41b0-8110-2e39c8e07c57)

What do you guys think. would this help you guys transition to using the W11 Taskbar?
I know this doesnt take care of side alignment or window grouping, but I can probably live with W11 like this and Never Hide, Never Combine. (as much as I would like Hide Labels, Never Combine).

**Comment by pyrates999** [comment]
You can disable system tray icons from not being shown like the sound icon: https://www.xda-developers.com/how-remove-icons-taskbar-windows-11/

I think what people want the most is to move the windows 11 taskbar to the sides and top, not just have it be on the bottom.

**Comment by I-Dude** [comment]
Thanks for the info, but I am referring to the System Tray Icons (the Battery, Network, Speaker) which brings up the control center.
There are no customization options for the W11 System Tray Icons. What I am proposing is allowing each of the icons to pull up a separate menu as it was in W10, instead of in W11 all three icons collectively pulling up the control center.

W11 23H2 control center:
![W11 Taskbar Control Center](https://github.com/valinet/ExplorerPatcher/assets/1747168/e75d5718-f10a-407a-8653-84e42e3d2e42)

**Comment by pyrates999** [comment]
Unless there is existing code allowing for that, then maybe.  Otherwise EP has no way to enable that.

**Comment by CHougland** [comment]
Can someone please make an .exe to separate the control panel icons, I've been upgraded to W11 for less than 72 hours and it's really pissing me off.

**Comment by vectorl04** [comment]
Finally I found someone who has same "issue" as me. I like the visual of W11 taskbar, but genuely hate the "wifi-sound..." box in one tile, I like it separated, as you said, less clicks. But I was unable to find sollution for it, unless to use W10 taskbar but it has worse visual than the W11 one. So separating the two icons would be awesome
